nessie/lib/metadata.py

```diff
@@ -30,6 +30,17 @@
 from nessie.externals import rds, s3
 
 
+def create_canvas_api_import_status(job_id, term_id, course_id, table_name):
+    sql = f"""INSERT INTO {_rds_schema()}.canvas_api_import_job_status
+               (job_id, term_id, course_id, table_name, status, instance_id, created_at, updated_at)
```

Comment:
@lyttam Does It make sense to include a column on what type of Canvas API we are importing from ?

Reply:
@sandeepmjay the `table_name` column will contain values such as 'gradebook_history' and 'grade_change_log' (referencing the tables in the canvas_api_ext_dev schema, but also identifying which API was used).

Reply:
sounds good. Thanks, @lyttam.